src/generate-config.ts

```diff
@@ -11,42 +11,6 @@ import { validateConfig } from "./utils/utils";
 import { buildData } from "./constants";
 const DIR = path.join(__dirname, "../build");
 
-// addresses exists as txt-records in respective domains
-const buildData = [
-  {
-    chainId: "51" as CHAIN_ID,
-    documentStoreAddress: "0x268852277C0eED5A9999B41b0FdbA0443De76475",
-    tokenRegistryAddress: "0x1a378fEEc3ed9B63B872B11561FCf19f6d2CE793",
-    dnsVerifiable: "tradetrust-apothem.xdc.network",
-    dnsTransferableRecord: "tradetrust-apothem.xdc.network",
-    dnsDid: "tradetrust-apothem.xdc.network",
-  },
```

Comment:
add this back to [`constants.ts`](https://github.com/TradeTrust/tradetrust-config/blob/87b918adf826f843cfb7ce3336394b0ba6091c64/src/constants.ts#L4), do a `npm run build` to ensure latest config files are built. do note that you did not provide your testnet wallet, so whatever configs generated here will not work anyway.

Reply:
Updated configs in constants.ts, Kindly review it now.

Reply:
can you check what's wrong with pipeline and try to rectify it https://github.com/TradeTrust/tradetrust-config/actions/runs/5654917417/job/15319027874?pr=52#step:5:9

![Screenshot 2023-07-26 at 10 19 08 AM](https://github.com/TradeTrust/tradetrust-config/assets/4774314/091d2201-5706-43cd-9c68-756d2a5571ba)